homeassistant/components/teslemetry/entity.py

```diff
@@ -192,3 +192,10 @@ def _value(self) -> int:
             .get(self.din, {})
             .get(self.key)
         )
+
+    @property
+    def has(self) -> bool:
```

Comment:
Is it possible to use a more meaningful name for this function? E.g. `has_xxxx()`?

Reply:
Its overloading an existing function called `has`, do you want me to change both or have this child class use a unique property?

Reply:
Ok to leave as is if overloading existing.